openstack/blockstorage/v1/snapshots/testing/requests_test.go

```diff
@@ -27,12 +29,20 @@ func TestList(t *testing.T) {
 
 		expected := []snapshots.Snapshot{
 			{
-				ID:   "289da7f8-6440-407c-9fb4-7db01ec49164",
-				Name: "snapshot-001",
+				ID:        "289da7f8-6440-407c-9fb4-7db01ec49164",
+				Name:      "snapshot-001",
+				VolumeID:  "521752a6-acf6-4b2d-bc7a-119f9148cd8c",
+				Status:    "available",
+				Size:      30,
+				CreatedAt: gophercloud.JSONRFC3339MilliNoZ(time.Date(2012, 2, 14, 20, 53, 7, 0, time.UTC)),
```

Comment:
Ach, ignore that ^; I'd just woken up. I see now it's from a unit test; I'll have a look.